src/Raven.Server/Documents/Handlers/TimeSeriesHandler.cs

```diff
@@ -527,9 +517,17 @@ public async Task ConfigTimeSeries()
                 GetRaftRequestIdFromQuery(),
                 beforeSetupConfiguration: (string name, ref BlittableJsonReaderObject configuration, JsonOperationContext context) =>
                 {
-                    if (configuration == null || 
-                        configuration.TryGet(nameof(TimeSeriesConfiguration.Collections), out BlittableJsonReaderObject collections) == false ||
-                        collections?.Count > 0 == false)
+                    if (configuration == null)
+                    {
+                        return;
+                    }
+
+                    var hasCollectionsConfig = configuration.TryGet(nameof(TimeSeriesConfiguration.Collections), out BlittableJsonReaderObject collections) &&
+                                               collections?.Count > 0;
+
+                    var hasNamedValuesConfig = configuration.TryGet(nameof(TimeSeriesConfiguration.NamedValues), out BlittableJsonReaderObject namedValues);
+                    
+                    if (hasCollectionsConfig == false && hasNamedValuesConfig == false)
```

Comment:
What if `hasCollectionsConfig == false` and `hasNamedValuesConfig == true` ? The code below can throw? Also are we doing something with `namedValues` variable ?